dev/com.ibm.ws.jpa.tests.spec10.entity_fat.common/test-applications/entity/src/com/ibm/ws/jpa/fvt/entity/testlogic/VersioningTestLogic.java

```diff
@@ -85,14 +85,7 @@ public void testVersioning001(TestExecutionContext testExecCtx, TestExecutionRes
 
         final boolean isDB2ZOS = DatabaseVendor.checkDBProductName(dbProductName, dbProductVersion, DatabaseVendor.DB2ZOS);
 
-        /*
-         * TODO: Disabling these tests until the issue is is delivered:
-         * https://github.com/OpenLiberty/open-liberty/issues/23949
-         * https://github.com/OpenLiberty/open-liberty/issues/23951
-         * https://github.com/OpenLiberty/open-liberty/issues/23952
-         */
-        if (isDB2ZOS && JPAPersistenceProvider.ECLIPSELINK.equals(provider)
-            && (isUsingJPA21Feature() || isUsingJPA30Feature() || isUsingJPA31Feature())) {
+        if (isDB2ZOS && JPAPersistenceProvider.ECLIPSELINK.equals(provider)) {
```

Comment:
Only the TODO for 23951 should be removed here, like the other ones below.
The test will pass in a build because this change effectively disables the test for all JPA levels when run with DB2ZOS,
Instead, we should just be removing the disable check for JPA30.

Reply:
Resolved